hack/generate-yaml.sh

```diff
@@ -100,6 +100,11 @@ echo "Generating volcano yaml file into ${DEPLOYMENT_FILE}"
 if [[ -f ${DEPLOYMENT_FILE} ]];then
     rm ${DEPLOYMENT_FILE}
 fi
+
+if [[ -f ${MONITOR_DEPLOYMENT_YAML_FILENAME} ]];then
```

Comment:
Why introduce monitor part in generate-yaml.sh

Reply:
I don't know Why introduce monitor part in generate-yaml.sh, this change is just a optimization， If Not， the operater in volcano-monitoring-latest.yaml is append and has the repeat data ; you have a look at it currently , you can find it